src/main/java/nextstep/subway/line/domain/Sections.java

```diff
@@ -57,6 +66,21 @@ public Stations getStations() {
         return stations;
     }
 
+    public Stations getStations() {
+        if (sections.isEmpty()) {
+            return Stations.emptyStations();
+        }
+
+        Stations stations = new Stations();
+
+        sections.stream()
```

Comment:
```suggestion
        return sections.stream()
            .flatMap(section -> Stream.of(section.getUpStation(), section.getDownStation()))
            .distinct()
            .collect(Collectors.collectingAndThen(Collectors.toList(), Stations::new));
```
Collectors.collectingAndThen를 쓰시면 바로 리턴하게 만들 수 있어요

Reply:
이렇게도 가능하군요. 하나 배웠습니다.